docs/guides/networking/ssh/advanced-ssh-tunneling/index.md

```diff
@@ -0,0 +1,216 @@
+
+2. This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server/) to create a standard user account, harden SSH access and remove unnecessary network services.
+
+3. When choosing ports for SSH tunneling, you should choose ports higher than 1024.  Ports below 1024 are reserved for specific protocols or root processes and may cause conflicts with your SSH process.
+
+This guide will use various IPs and ports that will change depending on the example, your configuration, and your needs. To best prevent confusion, the IP addresses will remain the same and are described below.
+
+- Your local machine, the ssh client: `127.0.0.1` or `localhost` 
+
+- The ssh server, which will serve as the pivot point: `10.10.10.2`
+
+- Destination machine, which is not publicly facing, but is accessible from the SSH server: `10.10.10.100` 
+
+- As many configurations require `sudo` permissions, I will be using the `root` user.
+
+{{< caution >}}
+Effective use of tunneling can allow you to bypass security measures such as firewall rules and gain access to internal services and machines.  This ability is also a double-edged sword, so it is important to disable access to these options in your SSH server configuration when you are not using them, to prevent malicious actors from accessing sensitive data.
+{{< /caution >}}
+
+## Preparing SSH for Tunneling
+
+1. Set your SSH configuration file to allow Port-Forwarding.  By default, it is located at `/etc/ssh/ssh_config`.  
+   
+       AllowTcpForwarding yes
+       AllowStreamLocalForwarding yes
+       GatewayPorts yes
+   
+   {{< note >}}
+   Possible values for `AllowTcpForwarding` are `yes`, `no`, `remote` (to allow remote forwarding) and `local` (to allow local forwarding)
+   
+   Possible values for GatewayPorts are `yes` (anyone can connect to remote forwarded ports), `no` (only local connections from server are allowed, this is the Default value) and `clientspecified` (which you would follow with a specific IP address, though if left blank, anyone can connect)
+   
+   {{< /note >}}
+
+2. Make sure the machines involved have at least three network interfaces (for example, `loopback`, `eth0` and `tun0`).
+   
+           ifconfig
+
+3. If you have local access to the SSH server, make sure `sshd` is running.
+   
+   ```bash
+    sudo systemctl start sshd
+   ```
+   
+   
+
+# Local Port Forwarding
+
+Local port forwarding allows a user to expose local, internal resources to a server.  The remote server can then access that resource quite transparently as if it was within its local network.  For example, a user may want to forward a port (and its attached service) to a corporate web server, an internal mail server's IMAP port, an SMB share, or almost any other service running on an internal network.  
+Local port forwarding is denoted by the `-L` option.
+
+```bash
+ ssh -L local_port:destination:destination_portbash

```

Comment:
In markdown, you can set syntax highlighting by putting the code language in this format into the code block.  but since y'all use 8 spaces for code blocks instead, I'll go ahead and pull these out